ui/public/i18n/en/translation.json

```diff
@@ -24,7 +24,32 @@
     "title": "Settings",
     "configure_instance": "Configure {instance}",
     "save": "Save",
-    "test_field": "Test field"
+    "receiver_emails": "Email notifications",
+    "receiver_emails_list": "List of email recipients for notifications",
+    "whitelists": "Allow list of CIDR, IP and Fully qualified domain name",
+    "receiver_emails_placeholder": "One email address per line",
+    "whitelist_placeholder": "One entry per line",
+    "advanced": "Advanced",
+    "ban_local_network": "Allow bans on the local network",
+    "ban_local_network_tips": "By default no ban occurs on private IP class",
+    "enabled": "Enabled",
+    "disabled": "Disabled",
+    "dyn_bantime": "Dynamic ban time",
+    "dyn_bantime_tips": "Ban time is calculated by ((number of attempts + 1) * 4) minutes",
+    "bantime": "Bantime",
+    "minutes": "Minutes",
+    "disable_online_api": "Disable central API",
+    "disable_online_api_tips": "The central API is used to retrieve blacklists and to have a glance on metrics at app.crowdsec.net",
+    "open_app_crowdsec": "App crowdsec website",
+    "enroll_instance": "Enroll the crowdsec instance",
+    "enroll_instance_must_be_real_token": "The token must be retrieved on the website app.crowdsec.net",
```

Comment:
could you try embedding a URL, I think it should work
```suggestion
    "enroll_instance_must_be_real_token": "The token must be retrieved on the website <a href='app.crowdsec.net'>app.crowdsec.net</a>",
```

Reply:
I do not like to add html inside translations, we will have error with translators one day, I added a button to reach the website, does it sound enough ?

Reply:
![image](https://user-images.githubusercontent.com/3164851/233782090-b6591051-8e9d-4863-af9f-97a7034e604c.png)

Reply:
Thanks, for me it's ok.